.github/workflows/deploy.yml

```
@@ -35,3 +35,5 @@ jobs:
           source ./.ci_scripts/update_docs
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REF: ${{ github.ref }}
+          GH_ACTOR: ${{ github.actor }}
```

Isn't this the user that triggered the CI (likely the person who merged). I think this gives you `conda-forge/conda-forge.github.io`:
```suggestion
          GH_ACTOR: ${{ github.repository }}
```

---

IDK. I was looking at this: https://github.community/t/dont-run-actions-on-pull-request-from-fork/17560

---

I changed this around a bit.